crates/uv/src/commands/tool/list.rs

```diff
@@ -99,5 +107,6 @@ pub(crate) async fn list(
         }
     }
 
+    writeln!(printer.stdout(), "\nSee `uvx --help` for more information.")?;
```

Comment:
I'd also move this to the `tool/run.rs` implementation and show "See `uv tool run --help`" when `uv tool run` is used instead of `uvx`.